From 717402232fb7e9588203a7aa99ed2b96feb35d2b Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Sat, 9 Jun 2018 17:53:25 +0200
Subject: [PATCH 2/3] klte-common: Reorder tetherable connection types

* In O, the default setting for mobile data always active was
  changed from off to on, meaning that when non-WiFi tethering
  is being attempted, more than one connection can be active
* Since the first active connection type in this overlay is used,
  reorder the values to get WiFi connections first, and then
  any type of mobile data connection, instead of vice versa

Change-Id: I41f31ba1a2793e209ff439c9ba28a390fecdeecb
---
 overlay/frameworks/base/core/res/res/values/config.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/overlay/frameworks/base/core/res/res/values/config.xml b/overlay/frameworks/base/core/res/res/values/config.xml
index b4589aa..874e465 100644
--- a/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/overlay/frameworks/base/core/res/res/values/config.xml
@@ -166,8 +166,8 @@
     <!-- Common options are [1, 4] for TYPE_WIFI and TYPE_MOBILE_DUN or
     <!== [0,1,5,7] for TYPE_MOBILE, TYPE_WIFI, TYPE_MOBILE_HIPRI and TYPE_BLUETOOTH -->
     <integer-array translatable="false" name="config_tether_upstream_types">
-        <item>0</item>
         <item>1</item>
+        <item>0</item>
         <item>5</item>
         <item>7</item>
     </integer-array>
-- 
2.17.1

